regions/io/fits/write.py

```diff
@@ -0,0 +1,59 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from astropy.io import fits
+
+from ..core import to_shape_list, SkyRegion
+
+
+__all__ = [
+    'write_fits_region',
+    'fits_region_objects_to_table',
+]
+
+
+def fits_region_objects_to_table(regions):
+    """
+    Converts list of regions to FITS region table.
+
+    Parameters
+    ----------
+    regions : list
+        List of `regions.Region` objects
+
+    Returns
+    -------
+    region_string : `~astropy.table.Table`
+        FITS region table
+
+    Examples
+    --------
+    TODO
+    """
+    for reg in regions:
+        if isinstance(reg, SkyRegion):
+            raise TypeError('This {} must be a pixel region'.format(reg))
+
+    shape_list = to_shape_list(regions, coordinate_system='image')
+    return shape_list.to_fits()
+
+
+def write_fits_region(regions, header=None, filename='new.fits'):
```

Comment:
`filename` should not have a default argument; make that a required argument instead of a kwarg

Reply:
Ah! Then, what about for `write_crtf` and `write_ds9`? They both have default filenames. Should we change them to required arg too?

Reply:
Yes, I think so!  I hadn't realized that.   It seems wrong to have a default filename, no?

Reply:
You are correct, it seems wrong. It would be better as a mandatory arg.